### ISSUE TITLE
Detected on battleEye

### ISSUE THREAD

**Issue by Wearwolfy** [rezadfx/HWID-shifter#1]
I tried this spoofer and it is detected on BattleEye unfortunatly

**Comment by rezadfx** [comment]
This project tested on BE at 21st Dec.2020 and was okay
Unfortunately or luckily Anticheat systems developing every day and proving their detection systems.

**Comment by Wearwolfy** [comment]
Yes no worries thanks alot, it's a really cool project though, just wanted to let you know :) but gj
